docs/api/build_proto.md

```diff
@@ -0,0 +1,346 @@
+            let mut ids: Vec<String> = Vec::new();
+            for v in vec.iter() {
+                let id = Utc::now().timestamp_nanos().to_string();
+                ids.push(id.to_string());
+                // insert vector
+                let _ = insert_client
+                    .insert(insert::Request {
+                        vector: Some(object::Vector {
+                            id: id.to_string(),
+                            vector: v.to_vec(),
+                        }),
+                        config: Some(insert::Config {
+                            skip_strict_exist_check: true,
+                            filters: None,
+                            timestamp: Utc::now().timestamp(),
+                        }),
+                    })
+                    .await?;
+            }
+            print!("[End] Finish Insert Phase\n");
+
+            print!("[Sleep] Waiting SaveIndex is completed...\n");
+            sleep(time::Duration::from_secs(DURATION));
+
+            print!("[Start] Search phase\n");
+            // create search client
+            let mut search_client =
+                search_client::SearchClient::connect(Endpoint::from_static(HOST)).await?;
+            for id in ids.clone() {
+                // search nearest neighbor vectors using searchById method
+                let res = search_client
+                    .search_by_id(search::IdRequest {
+                        id: id.to_string(),
+                        config: Some(search::Config {
+                            request_id: id.to_string(),
+                            num: 10,
+                            radius: -1.0,
+                            epsilon: -1.0,
+                            timeout: 500,
+                            ingress_filters: None,
+                            egress_filters: None,
+                        }),
+                    })
+                    .await?;
+                print!("[Id]: {:?}\n", id.to_string());
+                print!("[Result]: {:#?}\n", res.into_inner().results);
+            }
+            print!("[End] Finish Search Phase\n");
+
+            print!("[Start] Remove phase\n");
+            // create remove client
+            let mut remove_client =
+                remove_client::RemoveClient::connect(Endpoint::from_static(HOST)).await?;
+            for id in ids.clone() {
+                // remove vectors
+                let _ = remove_client
+                    .remove(remove::Request {
+                        id: Some(object::Id { id: id.to_string() }),
+                        config: None,
+                    })
+                    .await?;
+            }
+            print!("[End] Finish Remove phase\n");
+
+            Ok(())
+        }
+        ```
+
+1.  Running example
+
+    After creating Vald cluster, you can run an example code by following command.
+
+    ```bash
+    $ cargo run src/client.rs
```

Comment:
it fails to execute `cargo run src/client.rs`, I am not sure if it is caused by this workaround above.
https://github.com/vdaas/vald/pull/1540/files#r796392775

I restore back the `bin` section in `Cargo.toml` before executing this step.
```
❯ cargo run src/client.rs
   Compiling vald-grpc v0.1.0 (/Users/wdiu/go/src/github.com/vdaas/vald/vald-grpc)
error[E0433]: failed to resolve: use of undeclared crate or module `vald_sample_rust_client`
 --> src/client.rs:9:5
  |
9 | use vald_sample_rust_client::payload::v1::insert;
  |     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `vald_sample_rust_client`

error[E0433]: failed to resolve: use of undeclared crate or module `vald_sample_rust_client`
  --> src/client.rs:10:5
   |
10 | use vald_sample_rust_client::payload::v1::object;
   |     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `vald_sample_rust_client`

error[E0433]: failed to resolve: use of undeclared crate or module `vald_sample_rust_client`
  --> src/client.rs:11:5
   |
11 | use vald_sample_rust_client::payload::v1::remove;
   |     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `vald_sample_rust_client`

error[E0433]: failed to resolve: use of undeclared crate or module `vald_sample_rust_client`
  --> src/client.rs:12:5
   |
12 | use vald_sample_rust_client::payload::v1::search;
   |     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `vald_sample_rust_client`
......
```

Reply:
Did you add the `src/lib/rs` and success import your own protobuf?